src/app/shared/filter-bar/filter-bar.component.css

```diff
@@ -1,22 +1,67 @@
-.dropdown-filters .mat-form-field {
+.filters .mat-form-field {
   margin: 8px;
   font-size: 14px;
   max-width: 20%;
   width: 14%; /* depends on number of filters*/
 }
 
-.search-bar {
-  width: 80%;
+ul {
+  margin: 8px;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  list-style-type: none;
 }
 
-.dropdown-filters {
+.search-bar {
   width: 100%;
 }
 
+.filters {
+  width: 57%;
+  display: flex;
+}
+
 .label-filter-grid-tile {
   width: 100%;
 }
 
-.submit-button {
-  margin-left: 8px;
+.menu {
+  padding: 0 10px;
+  border-left: 1px solid #eee;
+  display: none;
+  cursor: pointer;
+
+  @media (max-width: 1148px) {
+    display: block;
+    justify-items: right;
+  }
+}
+
+.filters {
+  display: flex;
+  align-items: center;
+
+  @media (max-width: 1148px) {
+    display: none;
+
+    &.is-open {
+      display: block;
+      position: absolute;
+      width: 100%;
+      top: calc(20% + 8px); /* 8px margin from ul */
```

Comment:
Would it be better to use a calculation relative to the app-filter-bar? By applying a position: relative to the app filter bar you can position the child element of class filters relative to that parent when the child has position: absolute. Therefore you could then do something like top: 100%; instead on this element and it will be positioned nicely under the filter bar.
![image](https://github.com/user-attachments/assets/ec4c43b8-f865-4c0e-9809-6e1d609f5de4)

Reply:
Could you share your reasoning behind the 8px margin from ul as commented? I find it uneccessary and it would be better to associate the expansion more closely with the app-filter-bar.